dao-services/src/main/java/org/exoplatform/wiki/jpa/search/AttachmentIndexingServiceConnector.java

```diff
@@ -87,16 +87,10 @@ public Document create(String id) {
     Map<String,String> fields = new HashMap<>();
     Document doc = new Document(TYPE, id, getUrl(attachment), fileItem.getFileInfo().getUpdatedDate(),
         computePermissions(fileItem.getFileInfo().getUpdater(),attachment), fields);
-    int index = fileItem.getFileInfo().getName().lastIndexOf(".");
-    if(index != -1) {
-      doc.addField("title", fileItem.getFileInfo().getName().substring(0, index - 1));
-    }
-    else{
-      doc.addField("title", fileItem.getFileInfo().getName());
-    }
-
+    String fileName = org.exoplatform.wiki.utils.Utils.escapeIllegalCharacterInName(fileItem.getFileInfo().getName());
+    doc.addField("title", fileName);
```

Comment:
I think it's the same result, because this operation is used when computing the title of the attachement.
In fact, the title is computed here:
https://github.com/exoplatform/wiki/blob/develop/wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiUploadAttachment.java#L98
But, I think we can improve this by reusing the computed title, I will use the field "fullTitle" that is already computed in the UI Upload Controller.